guides/user-guide.md

```diff
@@ -0,0 +1,67 @@
+---
+expires: 2019-12-31
+---
+
+# Cyber Security Consulting Team: asking for help
+
+## Overview
+
+This document tells you about the Cyber Security Consulting Team, explains how to ask for help, outlines how your requests are handled, and describes what happens next.
+
+To ask for help from a cyber security consultant, send an email to:<br/>
+[cyberconsultancy@digital.justice.gov.uk](mailto:cyberconsultancy@digital.justice.gov.uk)
+
+## About the team
+
+The Cyber Security Consulting Team is part of Ministry of Justice Security & Privacy. The consultants are led by the MoJ Chief Information Security Officer, Jon Lawrence.
+
+The team provides help and guidance around cyber security matters, such as:
+
+- Understanding the risks facing your systems and services.
```

Comment:
...and designing and implementing effective mitigations to these risks.

Reply:
Added.